src/Bicep.Core/TypeSystem/IntegerLiteralType.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Bicep.Core.TypeSystem;
+
+public class IntegerLiteralType : TypeSymbol
+{
+    public IntegerLiteralType(long value)
+        : base(value.ToString())
```

Comment:
Unlikely to [have any practical impact](https://stackoverflow.com/questions/8492449/is-int32-tostring-culture-specific), but I think it's a good idea to get into the habit of using `CultureInfo.InvariantCulture`

```suggestion
        : base(value.ToString(CultureInfo.InvariantCulture))
```

Reply:
We should turn on the globalization analyzers at some point as well.